python/libs/Modifier.py

```diff
@@ -226,7 +226,7 @@ def copy_annotation_files(self):
     def copy_event_files(self):
         for eegRun in self.data.get('eegRuns'):
             eeg_file = eegRun['eegBIDSBasename']
-            filename = os.path.join(self.get_eeg_path(), eeg_file + '_annotations')
+            filename = os.path.join(self.get_eeg_path(), eeg_file + '_events')
```

Comment:
will this change the behaviour for annotation files ?

Reply:
I don't think so. There is a function specific for the annotations as well. I have a feeling it was a copy/paste but the person forgot to replace annotations with events. Unless I am misreading?

Reply:
yes, i think you're right.